### PR TITLE
Create environment to configure front-end

### PR DESCRIPTION
**Pull request by douwehoonhout**

### CODE REVIEW

File: web-application/src/certificates/certificate-creation.component.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class CertificateCreationComponent {
     delete = false;
 
     jsonThing = JSON.stringify;
-    states = [{type: 'ISSUED'}, {type: 'REVOKED'}];
+    states = [{type: 'ISSUED'}, {type: 'REVOKED'}, {}];
```
Comment (Harbar-Inbound) [inline web-application/src/certificates/certificate-creation.component.ts:24]: What does this empty object signify? Is it for deleted certificates?

**Review by Harbar-Inbound — COMMENTED** [review]
Looks ready to merge. The different port numbers need to be addressed in the readme/start guide. But that is an issue to squash this week

File: web-application/src/certificates/certificate-creation.component.ts (hunk 1 above)
Comment (douwehoonhout) [reply web-application/src/certificates/certificate-creation.component.ts]: Should have been for EXPIRED states whoops :)

File: smart-contracts/application-typescript/src/serverFarmer.ts [hunk 2]
```diff
@@ -11,15 +11,23 @@ app.use(bodyParser.urlencoded({extended: false}));
 
 export class ServerFarmer {
 
-    static port = 4100;
+    static port = 4101;
```
Comment (Harbar-Inbound) [inline smart-contracts/application-typescript/src/serverFarmer.ts:14]: The pstatic pot can be removed, from this class

File: web-application/package.json [hunk 3]
```diff
@@ -1,51 +1,54 @@
...
-    "karma-coverage": "2.0.3",
-    "karma-jasmine": "4.0.1",
-    "karma-jasmine-html-reporter": "1.5.4",
-    "protractor": "7.0.0",
-    "ts-node": "8.3.0",
-    "tslint": "6.1.3",
-    "typescript": "4.1.5"
-  }
+    "name": "blockcert-demo",
+    "version": "0.0.0",
```
Comment (Harbar-Inbound) [inline web-application/package.json:3]: Maybe update version number?

**Review by Harbar-Inbound — APPROVED** [review]
Two minor things, other than that it looks good!

**Pull request merged**